crates/core/tedge_api/src/health.rs

```diff
@@ -44,7 +44,7 @@ impl ServiceHealthTopic {
         }
     }
 
-    pub fn up_message(&self) -> Message {
+    pub fn up_message(&self, topic_root: &str) -> Message {
```

Comment:
Instead of a `&str`, it would be better to pass a `&MqttSchema`.

Reply:
using mqttschema